plugin/rename.py

```diff
@@ -24,10 +24,70 @@
 import sublime_plugin
 
 
+BUTTONS_TEMPLATE = """
+<style>
+    html {{
+        background-color: transparent;
+        height: 2rem;
+        padding-top: 0.3rem;
+    }}
+    a {{
+        display: inline;
+        line-height: 1.6rem;
+        padding-left: 0.6rem;
+        padding-right: 0.6rem;
+        border-width: 1px;
+        border-style: solid;
+        border-color: #fff4;
+        border-radius: 4px;
+        color: #cccccc;
+        background-color: #3f3f3f;
+        text-decoration: none;
+    }}
+    html.light a {{
+        border-color: #000a;
+        color: white;
+        background-color: #636363;
+    }}
+    a.primary {{
+        background-color: color(var(--accent) min-contrast(white 6.0));
+    }}
+    html.light a.primary {{
+        background-color: color(var(--accent) min-contrast(white 6.0));
+    }}
```

Comment:
If it's the same for both then just add to `a`.

Reply:
Could you write how you mean it? This is only for the Apply button `class='primary'`. I duplicated the rule because otherwise the specificity of `html.light a` is higher than `a.primary`.

Reply:
I didn't think about specificity. But you can still convince those two selectors using comma